Index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="Style.css">
+    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+</head>
+<body>
+
+ <header class="top-bar">
+  <a href="#" id="logolink"><h2 class="logoname">My Logo</h2></a>
+  <nav >
+    <img src="images\Union.png" alt="logo">
+  </nav>
+</header>
+ <section class="section-data">
+    <h1 class="myname">I’m Surya
+      Glad to see you!</h1>
```

Comment:
- [optional] It will be awesome if you can adjust this text spacing so that it doesn't overlay the background-image according to the design. Kindly take note that our ability to reproduce perfectly a given template shows how well we pay attention to details. 💁‍♂️